src/gatsby/on-render-body.js

```diff
@@ -48,29 +65,38 @@ export default ({ setHeadComponents }, pluginOptions) => {
     // TO DO - Warn about missing options
   }
 
-  let agent;
-  if (instrumentationType === 'lite') {
-    agent = liteAgent;
-  }
-
-  if (instrumentationType === 'pro') {
-    agent = proAgent;
-  }
+  const init = `
+    window.NREUM||(NREUM={});
+    NREUM.init={distributed_tracing:{enabled:false},privacy:{cookies_enabled:true}};
+  `;
 
-  if (instrumentationType === 'proAndSPA') {
-    agent = proAndSpaAgent;
-  }
+  const loader = getCdnFromInstrumentationType(instrumentationType);
 
   const configs = `
     ;NREUM.loader_config={accountID:"${options.accountId}",trustKey:"${options.trustKey}",agentID:"${options.agentID}",licenseKey:"${options.licenseKey}",applicationID:"${options.applicationID}"}
     ;NREUM.info={beacon:"${options.beacon}",errorBeacon:"${options.errorBeacon}",licenseKey:"${options.licenseKey}",applicationID:"${options.applicationID}",sa:1}
   `;
 
   setHeadComponents([
+    // Initialize NR namespace
+    <script
+      key="gatsby-plugin-newrelic-init"
+      dangerouslySetInnerHTML={{
+        __html: init
+      }}
+    />,
+
+    // Load aggregator
+    <script
```

Comment:
Will it now always be required to load from the CDN or will there be continued support in New Relic's future for a desire to have this baked into the HTML? Feels like a bummer to be forced into adding the extra network request.

Reply:
@passion4code Ideally, we'd offer both as options, though I much prefer the CDN approach.

There are pros/cons to each.

CDN:
- Version - Easy to customize by version as well as keep up to date
- Performance - Serving via a CDN can be situationally preferred vs. an extra network request, smaller html page size at the cost of one more "network request" (which should be cached after the first page load)

Baked into the HTML:
- Version - We need to find a way to avoid storing the multitude of agent versions in the repo if we're going to support this. I'll investigate. 
- Performance - One less network request, but larger html size